tags/tag-developer-experience/charter.md

```diff
@@ -1 +1,67 @@
+
+Our work involves identifying gaps, promoting best practices, and supporting CNCF projects and communities to enhance the productivity and satisfaction of developers working with cloud native technologies. We aim to bridge tools, platforms, and practices to create a cohesive and frictionless developer experience across the cloud native ecosystem.
+
+More specifically, we:
+
+* Provide guidance and thought leadership on improving developer workflows and experience in cloud native application development
+* Promote secure-by-default developer workflows by integrating security and compliance considerations into the developer experience
+* Collaborate with CNCF projects to surface and resolve developer experience challenges
+* Conduct studies, gathering feedback from end users and contributors, and providing recommendations
+* Facilitate interoperability and standardization across developer tools in the CNCF ecosystem
+* Act as a bridge to empower developers and project maintainers
+
+# Out of Scope
+
+* Prescriptive mandates on tooling choices or project roadmaps
+* Evaluating or endorsing specific commercial developer tools outside of CNCF projects
+* Replacing or duplicating efforts by other TAGs unless explicitly partnered
+
+# Goals
+
+* Develop shared vocabulary, metrics, and frameworks to evaluate DevEX
+* Provide a central forum for discussing the developer experience within CNCF
+* Help CNCF projects better understand and serve developer needs
+* Publish guidance and reference materials for effective developer workflows
+* Encourage cross-project collaboration to improve consistency and interoperability of tooling
+* Foster collaboration around integrating security and compliance into cloud native development workflows to reduce friction and improve adoption of best practices
+* Reduce barriers to entry and help democratize access to cloud native development by promoting inclusive, accessible, and low-friction developer experiences
+
+# Non-Goals
+
+* Acting as a certification or compliance body for DevEX tooling
+* Becoming a support forum for individual developer issues
+
+# Subprojects & Initiatives
+
+Potential subprojects or initiatives may be formed around specific themes such as:
+* [Cloud Native and OCI Compliant Inner-Loop Tooling & Packaging for AI Engineers](https://github.com/cncf/toc/issues/1740)
+
+These subprojects and initiatives will operate under the oversight of TAG: DevEx and will produce artifacts or recommendations relevant to their focus area.
+
+# Alignment with CNCF TOC
+
+TAG: DevEx will actively coordinate with the CNCF Technical Oversight Committee (TOC) to ensure our efforts align and complement their work. We will also engage with SIGs and CNCF project maintainers to understand how developer experience can be integrated into existing CNCF initiatives.
+
+# Get Involved
+
+Everyone is welcome to join TAG: DevEx. Our work is public and community-driven. To participate:
+
+* Join our meetings and mailing list
```

Comment:
Are there any links to these?

Reply:
Links: 
- Charter and TAG Page: https://github.com/cncf/toc/tree/main/tags/tag-developer-experience
- Mailing List: https://lists.cncf.io/g/cncf-tag-developer-experience/
- Slack: [Tag TAG Developer Experience Slack](https://cloud-native.slack.com/archives/C08KGCXB458)